splash/render_options.py

```diff
@@ -20,14 +20,15 @@ def __init__(self, data):
         self.data = data
 
     @classmethod
-    def fromrequest(cls, request):
+    def fromrequest(cls, request, max_timeout=defaults.MAX_TIMEOUT):
```

Comment:
I think it would be better to implement max-timeout as a usual member of RenderOptions. What do you think?